packages/app-mobile/root.tsx

```diff
@@ -873,12 +900,12 @@ class AppComponent extends React.Component {
 		};
 
 		const statusBarStyle = theme.appearance === 'light' ? 'dark-content' : 'light-content';
-
 		return (
 			<View style={{ flex: 1, backgroundColor: theme.backgroundColor }}>
 				<SideMenu
 					menu={sideMenuContent}
 					edgeHitWidth={5}
+					openMenuOffset={this.state.sideMenuWidth}
```

Comment:
SideMenuWidth is being used to assign the SideMenu component's openMenuOffset prop( which set's the width of the menu ). Last time I checked, I noticed it wasn't causing a rerender. I'll check it again though, I might have missed something.

Reply:
Hi @laurent22 I've updated the pr. The sidemenu rerenders without the key prop.

Reply:
Ok thanks for the update, this solution with openMenuOffset makes sense and should be more robust